sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -325,9 +325,9 @@ export class ManagementClient extends LinkEntity {
         timeoutInSeconds: getRetryAttemptTimeoutInMs(options.retryOptions) / 1000,
         delayInSeconds:
           options.retryOptions &&
-          options.retryOptions.retryInterval &&
-          options.retryOptions.retryInterval >= 0
-            ? options.retryOptions.retryInterval / 1000
+          options.retryOptions.delayInMs &&
+          options.retryOptions.delayInMs >= 0
+            ? options.retryOptions.delayInMs / 1000
```

Comment:
In the spirit of avoiding the null check for `retryOptions` we should initialize `options.retryOptions` to `{}` if no value was provided like we are doing in a couple of other places in this PR

Reply:
Addressed this and other comments, please see latest commit